plugin/test-keeper/plugin/test_checker.go

```diff
@@ -0,0 +1,60 @@
+package plugin
+
+import (
+	"github.com/sirupsen/logrus"
+	"github.com/arquillian/ike-prow-plugins/git"
+)
+
+type TestChecker struct {
+	log *logrus.Entry
+}
+
+const configFile = "test-keeper.yaml"
+
+func (checker *TestChecker) checkTests(pr *git.PullRequest) (string, string, error) {
+
+	matchers := loadMatchers(pr)
+
+	checker.log.Infof("Checking for tests")
+	files, e := pr.GetAffectedFiles()
+	if e != nil {
+		return "", "", e
+	}
+
+	for _, matcher := range matchers {
+		for _, file := range files {
+			checker.log.Infof("%q: %q", file.Name, file.Status)
+
+			if matcher.isTest(file.Name) {
+				return "success", "There are some tests :)", nil
```

Comment:
I was thinking about it, but consider the case when the level of checking tests is different based on a type of the PR - "fix" could have higher expectations than the other ones... Then I would be good to give the context - what is expected

Reply:
If we will need such feature I think this will be the right moment to change it to some more structured response. For the time being `bool` sounds to be more accurate than `string` used in the status later on.